auth-web/src/views/auth/AccountSettings.vue

```diff
@@ -172,7 +172,7 @@
                 role="listitem"
                 :to="transactionUrl"
                 data-test="transactions-nav-item"
-                v-if="isPremiumAccount || isSbcStaffAccount || isStaffAccount"
+                v-if="isPremiumAccount || isSbcStaffAccount || isStaffAccount || isBasicAccount"
```

Comment:
Hey Jia, these are all the account types we have in prod so you can remove this v-if

Reply:
okay, thanks!